### PR TITLE
Improve logging on `run` command error

### PR DESCRIPTION
**Pull request by guilload**

### CODE REVIEW

File: quickwit/quickwit-serve/src/lib.rs [hunk 1]
```diff
@@ -511,18 +524,23 @@ pub async fn serve_quickwit(
...
+            ] {
                 match index_manager.create_index(index_config, false).await {
                     Ok(_)
                     | Err(IndexServiceError::Metastore(MetastoreError::AlreadyExists(
                         EntityKind::Index { .. },
-                    ))) => Ok(()),
-                    Err(error) => Err(error),
-                }?;
+                    ))) => {}
+                    Err(error) => bail!("failed to create {index_name} index: {error}",),
```
Comment (guilload) [inline quickwit/quickwit-serve/src/lib.rs:542]: We could also just log the error here. WDYT?

**Comment by guilload** [comment]
@fulmicoton, friendly nudge.

File: quickwit/quickwit-serve/src/lib.rs (hunk 1 above)
Comment (fulmicoton) [reply quickwit/quickwit-serve/src/lib.rs:542]: Yes... A error log might be useful here too.

**Review by fulmicoton — APPROVED** [review]

File: quickwit/quickwit-serve/src/lib.rs (hunk 1 above)
Comment (fulmicoton) [reply quickwit/quickwit-serve/src/lib.rs:542]: ah... I don't have a clear opinion right now.

Comment (guilload) [reply quickwit/quickwit-serve/src/lib.rs:542]: Ok. I'll leave it as is for now. Also, all errors in `quickwit_serve` are logged in `main`.

**Pull request merged**